package.json

```diff
@@ -22,8 +22,8 @@
     "test:ci": "jest --ci --coverage --reporters=default --reporters=jest-junit"
   },
   "engines": {
-    "node": "12.22.x",
-    "npm": "6.x"
+    "node": "16.13.x",
+    "npm": "8.x"
```

Comment:
question(non-blocking): shouldn't we add yarn here as well?

Reply:
Good attention. I will provide a change for this. 👍